src/linter/messages.ts

```diff
@@ -529,4 +540,13 @@ export const MESSAGE_INFO = {
 		},
 	},
 
+	[MESSAGE.REPLACED_BOOTSTRAP_PARAM]: {
+		severity: LintMessageSeverity.Error,
+		ruleId: RULES["no-deprecated-api"],
+
+		message: ({name, replacement}: {name: string; replacement: string}) =>
+			`Bootstrap parameter '${name}' should be replaced with ${replacement}`,
```

Comment:
I would suggest to also put the replacement in single quotes.

Reply:
Done